frontend/src/components/FormRenderer/index.tsx

```diff
@@ -111,6 +111,19 @@ export const FormRenderer = ({
   };
 
   if ("edit" === formType) {
+    if (Array.isArray(query?.data?.data?.data)) {
+      const newInitialValues: { [key: string]: { [key: string]: string | number }[] } = {};
```

Comment:
this `data.data.data` could be replaced safely and readably using https://lodash.com/docs#get


```
let queryData = _.get(query, 'data.data.data', []);
...
if (queryData.length) {
...
  queryData.map(...);
 ```